Makefile

```diff
@@ -48,14 +48,15 @@ coverview: cover
 	xdg-open _build/test/cover/index.html
 
 edoc:
-	@$(REBAR) skip_deps=true doc
+	@$(REBAR) edoc
 
 help:
 	@echo "Please use 'make <target>' where <target> is one of"
 	@echo "  start             starts a test redis cluster"
 	@echo "  stop              stops all redis servers"
 	@echo "  travis-run        starts the redis cluster and runs your tests"
 	@echo "  travis-install    install redis from 'unstable' branch"
```

Comment:
Seems that the target travis-run dont even exists

Reply:
travis-run exists but not travis-install. Let's update (or delete) those when we get travis or github actions running.